src/pymodaq/control_modules/utils.py

```diff
@@ -373,5 +375,5 @@ def do_init(self, do_init=True):
         raise NotImplementedError
 
     def send_init(self):
```

Comment:
should also change this send_init signature here with a checked boolean argument

Reply:
I'll do it

Reply:
Done